### ISSUE TITLE
Advanced Pass

### ISSUE THREAD

**Issue by jacob-davenport** [bafolts/terraforming-mars#6952]
Please add an option when it's not your turn to say that you will pass on your next turn.  This cool feature speeds up asynchronous games.

**Comment by kberg** [comment]
I wish. It would require a database restructuring. That kind of change is
underway, but it won't be for a while.

**Comment by jacob-davenport** [comment]
Fair enough!  Leave this way below in your backlog.  And thank you for building this.
